src/main/java/com/gregtechceu/gtceu/data/recipe/misc/WoodMachineRecipes.java

```diff
@@ -349,34 +350,35 @@ public static void registerWoodUnificationInfo(@NotNull WoodTypeEntry entry) {
      */
     public static void registerWoodTypeRecipe(Consumer<FinishedRecipe> provider, @NotNull WoodTypeEntry entry) {
         final String name = entry.woodName;
+        final Item[] logs_ = entry.getLogs();
 
         // noinspection ConstantValue can be null if someone does an oopsie and doesn't set it.
         if (entry.planks == null) {
             throw new IllegalStateException("Could not find planks form of WoodTypeEntry '" + name + "'.");
         }
 
         // log-associated recipes
-        for (var log_ : entry.getLogs()) {
-            if (log_ != null) {
+        for (int i = 0; i < logs_.length; i++) {
+            if (logs_[i] != null) {
                 // nerf regular log -> plank crafting, if enabled
                 boolean hasPlanksRecipe = entry.planksRecipeName != null;
                 if (ConfigHolder.INSTANCE.recipes.nerfWoodCrafting) {
                     VanillaRecipeHelper.addShapelessRecipe(provider,
-                            hasPlanksRecipe ? entry.planksRecipeName : name + "_planks",
-                            new ItemStack(entry.planks, 2), log_);
+                            hasPlanksRecipe ? entry.planksRecipeName : name + "_planks_" + i,
```

Comment:
could you instead format them like the vanilla log recipe ids?

Reply:
You mean format the cutter recipe ids like the vanilla log recipe ids?

Reply:
Oh I understood

Reply:
Wait, I don't understand. So do you want me to change this: `hasPlanksRecipe ? entry.planksRecipeName : name + "_planks_" + i` to `hasPlanksRecipe ? entry.planksRecipeName : name + "_planks"` or do you want me to change the ids of saw and cutter like the vanilla log recipe id? I think the first one right?

Reply:
actually, it should take in a tag like `minecraft:<name>_logs` instead of an array of possible logs.

Reply:
& like that, only have 1 recipe per.

Reply:
Yes, but this still could work. If we add a postfix to the recipe id, but add nothing when index is 0, it can still overwrite the original recipe.

Reply:
no. use the tag. so that if someone adds a new acacia log, it also works.

Reply:
Do you think that's a good idea or should we do tags instead

Reply:
Okay I will try to figure that out

Reply:
Fixed now Recipes use `<modid>:<name>_logs`